eng/pipelines/runtime-codeql.yml

```diff
@@ -0,0 +1,68 @@
+trigger:
+  none
+
+schedules:
+  - cron: 0 12 * * 1
+    displayName: Weekly Monday CodeQL/Semmle run
+    branches:
+      include:
+      - main
+    always: true
+
+variables:
+- template: /eng/pipelines/common/variables.yml
+# TODO: (Consolidation) Switch away from old signing/validation variables from former Core-Setup. https://github.com/dotnet/runtime/issues/1027
+- name: TeamName
+  value: dotnet-core-acquisition
```

Comment:
We should consolidate these if possible. The variables yml is likely a good one to hold them and also, we can probably fix the team name :)

Reply:
Consolidate as in just move `TeamName` and `PostBuildSign` to `variables.yml` because they exist in both `runtime-official` and `runtime-codeql` yml? It doesn't seem like many other `.yml` files that leverage `variables.yml` use those two variables. And I'm also thinking that the codeql variables would just be specific to this pipeline, is there more value to moving those to variables.yml instead? It feels like it would just clutter that template.

What would be a better team name? 
When I'm looking more closely at when TeamName is used, it looks like its just for MicroBuild signing and cleanup. Not sure how relevant/necessary that is for runtime-codeql, so perhaps `TeamName` can be outright removed.

Also looking at `PostBuildSign`, it doesn't seem necessary to have signing validation for this pipeline either?

Reply:
Those are the important ones to have that. `dotnet-runtime` is a good name - it's a crumb to get back to the depo for governance purposes. The QL variables should stay there. `PostBuildSign` gets used for more than that, but since it also doesn't do signing, team name and PostBuildSign are not needed. They should be set in general for internal pipelines, so I think moving off to the centralized ones under an internal check sounds good.

Reply:
Removed `TeamName` and `PostBuildSign` from `runtime-codeql.yml` and created an `internal-variables` template thats now used in `runtime-official.yml`